examples/stage/04_odc_k8s_sandbox/data_providers.tf

```diff
@@ -32,6 +32,11 @@ data "aws_eks_cluster_auth" "cluster" {
   name = data.terraform_remote_state.odc_eks-stage.outputs.cluster_id
 }
 
```

Comment:
I don't think the example is self-contained with this change - I couldn't find a place where the sandbox_db_ro_creds were uploaded to the aws_ssm_parameter store.
The example should run out of the box, so either use the state look up or have the appropriate tf in place to add this to the parameter store.